src/assessments/types/requirement.ts

```diff
@@ -55,6 +55,6 @@ export interface VisualHelperToggleConfig {
     assessmentNavState: AssessmentNavState;
     instancesMap: DictionaryStringTo<IGeneratedAssessmentInstance>;
     actionMessageCreator: DetailsViewActionMessageCreator;
-    isStepEnabled: boolean;
-    isStepScanned: boolean;
+    isRequirementEnabled: boolean;
```

Comment:
This is a breaking change to our persisted store data format. We can think about how to manage this, but it's a MUCH more frightening change than updating type and file names. Can you please revert the property rename from this PR to be considered separately?

Reply:
Good point. But VisualHelperToggleConfig seems not being used in stores or in background. It is only used in views. Let me know if I'm misunderstanding.

Reply:
I'm sorry, you're right; I saw `isStepScanned` in the persisted data and got concerned, but it comes from the property by that name in `manual-test-status.ts`, not from here. Still, I'd rather keep PRs this big to single mechanical refactorings at a time

Reply:
Cool, thanks.